modules/ROOT/pages/upgrade.adoc

```diff
@@ -4,88 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 
 == Before You Begin
-Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.
-
-Refer to the xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypoint Platform Private Cloud Edition Release Notes] for additional information about upgrading Anypoint Platform PCE to the most recent version.
-
-[WARNING]
-You must contact MuleSoft professional services prior to upgrading. MuleSoft professional services must be involved with your upgrade procedure.
-
-== Procedure
-
-The following steps describe how to upgrade Anypoint Platform PCE and the Anypoint Platform PCE infrastructure from v2.0.x to v2.1.0. 
-
-If you are upgrading from a prior version of Anypoint Platform PCE, you must first upgrade to v2.0.4 and then upgrade to v2.1.0. 
-
-[NOTE]
-====
-* During an upgrade, all Anypoint Platform PCE components are updated.
-* The upgrade process causes downtime for each node as it is updated.
-====
-
-. Perform a backup of your system as described in xref:backup-and-disaster-recovery.adoc[Configure Backup and Restore]. If you are upgrading from an older version that requires multiple intermediate upgrades, you must backup each intermediate version before proceeding to the next.
-
-. Obtain a link to the installer from your customer success representative.
 
-. Use `ssh` to log in to the master node of your cluster.
-+
-In a 3-node environment, all nodes are master nodes. To find the master nodes in a 6-node environment, run the following command:
-+
-----
-kubectl get node --show-labels | grep gravitational.io/k8s-role=master
-----
-+
-By performing these procedures from the master node, the upgrade scripts are able to update the entire platform.
+[IMPORTANT]
```

Comment:
Similar to the install page, we could potentially make this its own section for emphasis.

### MuleSoft Professional Services Required
MuleSoft professional services must be involved throughout the installation process. 

Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.

Reply:
Got it - thanks!